PathPlanning/WavefrontCPP/wavefront_coverage_path_planner.py

```diff
@@ -0,0 +1,200 @@
+"""
```

Comment:
Could you please add a unittest for this script like: https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_based_sweep_coverage_path_planner.py